docs/peopleops/onboarding/onboarding-process.md

```diff
@@ -0,0 +1,35 @@
+# Onboarding process
+
+Onboarding is incredibly important at Thoth Tech. We don't expect you to hit the ground running from day one.
+
+We highly recommend taking at least one full day for onboarding. Please feel free to participate in your team's
+work in your first two weeks, but don't feel like you are under heavy pressure. We are here to help.
+
+## Accounts & access
+
+- [ ] Submit the [onboarding form](https://teams.microsoft.com/l/entity/81fef3a6-72aa-4648-a763-de824aeafb7d/_djb2_msteams_prefix_1382414451?context=%7B%22subEntityId%22%3Anull%2C%22channelId%22%3A%2219%3AQfx_STHU90OsVYBHVYKhsRQ5gmEe0s9Q6kOpBf6bli81%40thread.tacv2%22%7D&groupId=0e15669c-3f66-49aa-b023-640fe1dda2e0&tenantId=d02378ec-1688-46d5-8540-1c28b5f470f6)
+- [ ] Set up a GitHub [account](https://docs.github.com/en/get-started/signing-up-for-github/signing-up-for-a-new-github-account) (if you do not have one currently)
+- [ ] Add your contact details in this [doc](https://deakin365.sharepoint.com/:x:/r/sites/ThothTech2/Shared%20Documents/General/Administration/Personal%20Contacts.xlsx?d=w279b188c84544b269712dc4d2a6f7b87&csf=1&web=1)
+- [ ] Ping a person in the leadership team to add you in the Thoth Tech organisation on GitHub
+- [ ] Sign up for an account using your Deakin email address in [Miro](https://miro.com/contact/education/).
+      We use this tool to do online collaboration. We will later create different projects and invite you.
+- [ ] Join the Trello board of your assigned projects.
+
+## Products
+
+We have two main products at Thoth Tech: OnTrack and SplashKit. Depending on which team you are working
+with, you are expected to have some good level of understanding about the product and how to set it up.
+
+- [ ] See OnTrack onboarding
+- [ ] See SplashKit onboarding
```

Comment:
**note:** I will link these two items to the corresponding markdown when they are reviewed and merged.

Reply:
I have linked the docs in the subsequent commit.